src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -551,6 +558,20 @@ static ConfigDef newConfigDef() {
             6,
             ConfigDef.Width.NONE,
             SNOWPIPE_STREAMING_MAX_CLIENT_LAG)
+        .define(
+            SNOWPIPE_STREAMING_CLIENT_PARAMETER_OVERRIDE_MAP,
+            Type.STRING,
+            "",
+            STREAMING_CLIENT_PROVIDER_PROPERTIES_VALIDATOR,
+            Importance.LOW,
+            "Map of Key value pairs representing Streaming Client Properties to Override (optional"
+                + " and recommended to use only after consulting Snowflake Support). Format :"
```

Comment:
Nit, I'd remove the parens and change to `Override. These are optional and recommend to use ONLY after consulting with Snowflake Support.`